sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -1,12 +1,11 @@
 <template>
   <modal-dialog-base title="Edit application" :showDialog="showDialog">
     <template #content>
-      Editing your application will result in a new assessment, which could
-      delay your application. <strong>Are you sure you want to proceed?</strong>
```

Comment:
Please double check that `Are you sure you want to proceed?` not not required to be bold.

Reply:
I have confirmed with Jeremy that there is no bolded text needed here.